udata/auth/__init__.py

```diff
@@ -38,22 +38,28 @@ def __init__(self, *needs):
 admin_permission = Permission()
 
 
-def init_app(app):
+def init_security(app, **kwargs):
     from .forms import ExtendedRegisterForm, ExtendedLoginForm, ExtendedResetPasswordForm
     from .tasks import sendmail_proxy
     from .views import create_security_blueprint
     from .password_validation import password_validator
     from udata.models import datastore
+
     state = security.init_app(app, datastore,
                               register_blueprint=False,
-                              login_form=ExtendedLoginForm,
-                              confirm_register_form=ExtendedRegisterForm,
-                              register_form=ExtendedRegisterForm,
-                              reset_password_form=ExtendedResetPasswordForm,
-                              send_mail=sendmail_proxy
-                            )
+                              login_form=kwargs.get('login_form') or ExtendedLoginForm,
+                              confirm_register_form=kwargs.get('confirm_register_form') or ExtendedRegisterForm,
+                              register_form=kwargs.get('register_form') or ExtendedRegisterForm,
+                              reset_password_form=kwargs.get('reset_password_form') or ExtendedResetPasswordForm,
+                              send_mail=sendmail_proxy)
```

Comment:
Could we try something like:

```python
default_kwargs={
    'register_blueprint': False,
    'login_form': ExtendedLoginForm,
    'send_mail': sendmail_proxy
    [...]
}

state = security.init_app(app, datastore, **{**default_kwargs, **kwargs})
```

Or something with `{}.update()` since that makes a lot of `**` 😅